src/main/java/org/samo_lego/golfiv/mixin/duplication/ServerPlayerEntityMixin_NoConnectionDeath.java

```diff
@@ -25,8 +26,8 @@ public abstract class ServerPlayerEntityMixin_NoConnectionDeath {
     /**
      * Checks whether player is even connected before applying damage.
      */
-    @Inject(method = "damage(Lnet/minecraft/entity/damage/DamageSource;F)Z", at = @At("HEAD"), cancellable = true)
-    private void damage(DamageSource source, float amount, CallbackInfoReturnable<Boolean> cir) {
+    @Inject(method = "damage", at = @At("HEAD"), cancellable = true)
+    private void damage(ServerWorld world, DamageSource source, float amount, CallbackInfoReturnable<Boolean> cir) {
```

Comment:
Admittedly this would be better to just reduce the signature to just the `cir`.
We don't need the rest.